src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:35291/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:35291/project/issues?id=sample&branch=feature1&issues=AWCcTMGkFgyty_M6Z969&open=AWCcTMGkFgyty_M6Z969)

[ISSUE_KEY]: AWCcTMGkFgyty_M6Z969